Puc/v4p2/Vcs/GitLabApi.php

```diff
@@ -0,0 +1,274 @@
+<?php
+
+if ( ! class_exists( 'Puc_v4p2_Vcs_GitLabApi', false ) ) :
```

Comment:
Code style: This project doesn't follow the WordPress coding standards on space usage. To stay consistent with the rest of PUC code, please don't put spaces around the list of function arguments and unary operators. This line should look like this:
```php
if ( !class_exists('Puc_v4p2_Vcs_GitLabApi', false) ):
```
There are a bunch of other function calls with too many spaces.